includes/Generator/Order.php

```diff
@@ -58,6 +58,28 @@ public static function generate( $save = true, $assoc_args = array() ) {
 		$order->set_shipping_postcode( $customer->get_shipping_postcode() );
 		$order->set_shipping_state( $customer->get_shipping_state() );
 		$order->set_shipping_country( $customer->get_shipping_country() );
+
+		if ( self::$faker->randomFloat( 0, 1, 5 ) == 1 ) {
```

Comment:
Minor suggestion for readability
```suggestion
		// 20% chance
		if ( rand( 0, 100 ) <= 20 ) {
```

Reply:
@michi-dev would you mind committing this suggestion?

Reply:
@Luc45 Yes no problem :)